mlpp_lib/value_filler.py

```diff
@@ -0,0 +1,32 @@
+import logging
+from dataclasses import dataclass, field
+from abc import abstractmethod
+
+import numpy as np
+import xarray as xr
+from typing_extensions import Self
+
+
+class ValueFiller:
```

Comment:
I'd use "Imputer" since imputation is a widely used concept in ML, see e.g. https://scikit-learn.org/stable/modules/impute.html